custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -194,6 +194,32 @@
         "type": "raw",
         "property_name": "referral_reached_date"
       },
+      {
+        "column_id": "referral_reached_facility",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "referral_reached_facility"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
```

Comment:
is this going to require a rebuild?

Reply:
Also this table is not used in any active reports, so the consequences of a rebuild would not affect any AWW